src/plugins/data_views/common/data_views/persistable_state.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { DataViewBaseSerializable } from '@kbn/es-query';
+import { PersistableStateService } from '@kbn/kibana-utils-plugin/common';
+
+export const DataViewPersistableStateService: PersistableStateService<DataViewBaseSerializable> = {
+  inject: (state, references) => {
```

Comment:
As data view spec can contain fields we should call field methods inside each of this. 

cc @mattkime @sixstringcode if we don't plan to work on local runtime fields (if local data views can satisfy that requirement for some time) then we don't need to implement persistable state interface for fields.

Reply:
I believe from my conversation with @mattkime that this is still up in the air, so I'll remove the implementation for fields from this PR.